### ISSUE TITLE
Mark functions designed for verbose logging

### ISSUE THREAD

**Issue by Brett208** [OutpostUniverse/NetFixClient#65]
I'd like to add a LogVerbose function to logger.h/.cpp.

Then I would like to review the logging functions and sort them by marking them as Log or LogVerbose. This should allow removing all the // ** DEBUG ** lines. It will serve to mark which logging we think we want in the final product as opposed to functions that are logging just for debugging/test purposes.

For now, I'd like to add a hardcoded bool isVerbose to logger.cpp. We can use to turn on/off verbose logging manually in the source code. Once we determine how this should really be done, it can be updated into something more formal or removed if there is no such thing as verbose logging in the final product.

**Comment by DanRStevens** [comment]
I'm really tempted to say no to this idea, just to force more deep thinking on the problem.

----

I should point out that I used to think of logging as a messy debug artifact. Since then I've come to realized logging is often an important feature in a released product. Much of what I logged before may be appropriate, provided it doesn't become spammy. (Some of it was, mostly the commented out stuff).

You are free to take great liberties with what is logged, and the logging format. I might recommend doing that first before considering if verbose logging is even needed. If something is useful, log it. If something is spammy and not useful, don't log it. If something is spammy but sometimes useful, change when you log it.

**Comment by Brett208** [comment]
@DanRStevens, I would like one of us to start sorting the current log messages within NetFixClient to use the new choices on the public portion of op2ext's module API:

```C++
OP2EXT_API void Log(const char* message);
OP2EXT_API void LogError(const char* message);
OP2EXT_API void LogDebug(const char* message); 
```

Any issues with that?

**Comment by Brett208** [comment]
Closed by #85 

Still room to discuss in future issues if some of debug logging should be eliminated.
